pulsar-common/src/test/java/org/apache/pulsar/common/util/collections/ConcurrentLongHashMapTest.java

```diff
@@ -213,6 +214,66 @@ public void testExpandShrinkAndClear() {
         assertTrue(map.capacity() == initCapacity);
     }
 
+    @Test
```

Comment:
```suggestion
    @Test(invocationCount = 10)
```

It's better to add `invocationCount` to avoid regression because even if new changes breaks the behavior, the test might also get passed.

Reply:
done.